datafusion/sqllogictest/test_files/information_schema.slt

```diff
@@ -150,6 +150,7 @@ datafusion.execution.aggregate.scalar_update_factor 10
 datafusion.execution.batch_size 8192
 datafusion.execution.coalesce_batches true
 datafusion.execution.collect_statistics false
+datafusion.execution.ignore_subdirectory true
```

Comment:
Could we use a name that gives some context about when the `ignore_subdirectory` is actually used?

For example, maybe like `listing_table_ignore_subdirectory`. Or maybe even it is time to create a whole new category of configuration for listing tables `datafusion.listing_table.ignore_subdirectory` 🤔

Reply:
I used `listing_table_ignore_subdirectory`